apps/web/lib/linkValueToString.ts

```diff
@@ -39,3 +39,30 @@ export const linkValueToString = (
       return linkValue || "";
   }
 };
+
+export const integrationLocationToString = (location: string): string => {
+  switch (location) {
+    case "integrations:google:meet":
+      return "Google Meet";
+    case "integrations:daily":
+      return "Cal Video";
+    case "integrations:zoom":
+      return "Zoom";
+    case "integrations:huddle01":
+      return "Huddle01";
+    case "integrations:tandem":
+      return "Tandem";
+    case "integrations:office365_video":
+      return "MS Teams";
+    case "integrations:jitsi":
+      return "Jitsi";
+    case "integrations:whereby_video":
+      return "Whereby";
+    case "integrations:around_video":
+      return "Around";
+    case "integrations:riverside_video":
+      return "Riverside";
+    default:
+      return location;
+  }
+};
```

Comment:
@zomars is this something we wanna do? maybe there is a better way to return the name of the integration. I feel like this will throw errors once we add another Location-based App and forget to add it here.

I really don't want to have app-specific code spread out into the core code base

Reply:
> I feel like this will throw errors once we add another Location-based App and forget to add it here.

It shouldn't throw an error, it should just use the default location which then would be the unformatted location string